transformer.py

```
@@ -0,0 +1,276 @@
+
+# @title 1. 환경 설정 및 라이브러리 설치 (Cell 1)
+"""
+이 셀에서는 실습에 필요한 라이브러리들을 설치하고 임포트합니다.
+- transformers: Hugging Face 모델 및 파이프라인 사용을 위한 라이브러리
+- datasets: KLUE 데이터셋 등 Hugging Face Hub의 데이터셋 로드를 위한 라이브러리
+- sentencepiece: KoBART 등 일부 모델에서 사용하는 토크나이저 라이브러리
+- accelerate: 모델 로딩 및 분산 처리를 도와주는 라이브러리 (특히 NLLB 모델에 유용)
+- torch: PyTorch 라이브러리 (기본 백엔드)
+"""
+# TODO: 필요한 라이브러리를 설치하는 명령어를 작성하세요. (transformers, datasets, sentencepiece, accelerate, torch)
+get_ipython().system('pip install transformers datasets sentencepiece accelerate torch')
```

Ruff (0.13.3)

Undefined name `get_ipython`

(F821)